addons/l10n_din5008_sale/report/din5008_sale_templates.xml

```diff
@@ -94,5 +94,19 @@
                 </span>
             </t>
         </xpath>
+                    <!-- Insert Position Column as the First Column in the Table Header -->
+            <xpath expr="//table/thead/tr/th[1]" position="before">
+                <t t-if="user.env['ir.config_parameter'].sudo().get_param('account.display_position_column') == 'True'">
```

Comment:
no need for == 'True' the get_param will give you True or False depending if it finds the value i think

Reply:
As it is string so it see it true in care of 'False' or 'True' so both cases show the position column

Reply:
It is a string. Both `'True'` and `'False'` are truthy.
However, it might be nicer to use `str2bool` so that we understand `y`, `yes`, `1`, `true`, `t`, `on` and the opposite (case insensitive)